platformio.ini

```diff
@@ -0,0 +1,55 @@
+; Reference https://docs.platformio.org/en/latest/platforms/espressif32.html
+[platformio]
+default_envs = m5stick-c
+src_dir = alpha
+
+; Keep sensitive data in environment variables
+;
+; Unix
+; export WIFI_SSID='\"my\ ssid\ name\"'
+; export WIFI_PASS='\"my\ password\"'
+;
+; Windows
+; set WIFI_SSID='"my ssid name"'
+; set WIFI_PASS='"my password"'
+
+[common]
+; DEBUG Levels: 0=None, 1=Error, 2=Warn, 3=Info, 4=Debug, 5=Verbose
+debug_level = 3
+lib_deps_external =
+    ArduinoJson
+framework = arduino
+build_type = debug
+
+
+[env:m5stick-c]
+board = m5stick-c
+platform = https://github.com/platformio/platform-espressif32.git
+platform_packages =
+    framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git
+framework = ${common.framework}
+
+upload_speed = 1500000
+
+; Library options
+lib_deps =
+    ${common.lib_deps_external}
+    https://github.com/bettr-xyz/M5StickC.git
+	
+
+; change microcontroller
+board_build.mcu = esp32
+
+; change MCU frequency
+board_build.f_cpu = 240000000L
```

Comment:
I wonder if changing this could help to decrease cpu freq. Currently we're using 80mhz, yet to try 40mhz.

Reply:
perhaps, can look into it in the future, I'll just push this first then can make changes along the way, but Arduino setup will still be baseline